src/assets/scss/pages/story.scss

```diff
@@ -25,7 +25,7 @@
   }
 
   &-page {
-    height: 100vh;
+    height: 100%;
```

Comment:
100% height is only taking the 100% for this element in particular. 100vh takes the 100% from the viewport. That's the main difference here. We need the 100vh or a massive workaround that wouldn't make sense to be honest.

![image](https://user-images.githubusercontent.com/5013096/160702222-e4b969ef-4c24-4b8a-a8da-bb1394e93b2f.png)

Reply:
I've tested this myself, and I can give you the solution to what you're looking for:

use: `min-height: calc(100vh - 150px);`  and remove `height`